db/migrations/20200526095904_rebuild.js

```diff
@@ -14,7 +14,7 @@ function dropTables(knex) {
 
 function buildOrganization(knex) {
   return knex.schema.createTable('organizations', function (table) {
-    table.uuid('id').notNull().primary();
+    table.increments('id').notNull().primary();
```

Comment:
Can we also add a `completed_onboarding` flag (defaults to `false`) to the `organizations` table? If so, it'll knock out [this ticket](https://pathcheck.atlassian.net/browse/PLACES-244) 😉

Reply:
Done